.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,34 @@
+name: build-and-push
+
+on:
+ release:
+   types:
+     - published
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: ./
+        file: ./Dockerfile
+        push: true
+        tags: ioos/ioos-us:${{ github.event.release.tag_name }}
```

Comment:
I can also update this line to be - if the Dockerhub repo name matches the github repo
`tags: ioos/${{ github.event.repository.name }}:${{ github.event.release.tag_name }}`